src/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -184,7 +183,7 @@ public static void loadNationLevelConfig() throws IOException {
 						Integer.parseInt(level.get("nationBonusOutpostLimit").toString())
 						);
 			} catch (Exception e) {
-				Towny.getPlugin().getLogger().warning("Your Towny config.yml's nation_level section is out of date.");
+				Towny.getPlugin().getLogger().warning("The nation_level section of your Towny config.yml is out of date.");
```

Comment:
You may also want to change the grammar in the similarly phrased town-levels warning.

Reply:
Good eye.